<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectUnifiedCSVFolderDiag</class>
 <widget class="QDialog" name="SelectUnifiedCSVFolderDiag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Unified CSV</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select folder to process</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lePath">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbBrowse">
         <property name="text">
          <string>Browse ...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Tolerance for using medical records (in days)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leMedRecsDays">
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="text">
        <string>60</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Select which ID to use</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbIDToUse">
       <item>
        <property name="text">
         <string>(DID) Display ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(DBPUID) ViewMindDB ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(HPUID) ViewMind Internal Hashed ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(PUID) Viewmind Internal ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Name and Lastname</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pbOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
